securedrop/models.py

```diff
@@ -403,7 +407,7 @@ class Journalist(db.Model):
     is_admin = Column(Boolean)  # type: Column[Optional[bool]]
     session_nonce = Column(Integer, nullable=False, default=0)
 
-    otp_secret = Column(String(16), default=pyotp.random_base32)
```

Comment:
latest version of pyotp (2.6.0) now defaults to a 32-char base32 string from random_base32(). This should definitely be updated but it might make sense to update the dependency and the relevant references in the codebase. Any thoughts on that approach @evilaliv3 ?

Reply:
Thank you for noticing this @zenmonkeykstop , you are right the size of this string should be changed accordingly.

I actually notice to have to do this on our codebase as well and as a fun fact SQLITE seems to ignore the size limit (probably due to default allogators on Strings that always allocate more than just 16 chars)

Reply:
yup varchars are humongous, had some "fun" last night writing db migration tests for what is effectively a no-op.